borningssl/plan.sh

```diff
@@ -0,0 +1,59 @@
+pkg_name=boringssl
+pkg_origin=core
+pkg_description="BoringSSL is a fork of OpenSSL that is designed to meet Google\'s needs."
+pkg_upstream_url='https://boringssl.googlesource.com/boringssl/'
+pkg_version=3359
+pkg_maintainer='The Habitat Maintainers <humans@habitat.sh>'
+pkg_license=('OpenSSL ISC')
+pkg_source="https://boringssl.googlesource.com/boringssl"
+pkg_deps=(
+  core/glibc
+  core/git
+  core/go
+  core/cmake
+  core/ninja
+  core/perl
+  core/gcc
+)
+pkg_build_deps=(
+  core/glibc
+  core/gcc-libs
+)
+pkg_lib_dirs=(lib)
+pkg_include_dirs=(include)
+
+do_download() {
+  rm -rf "${HAB_CACHE_SRC_PATH}/${pkg_dirname}"
+  git clone "${pkg_source}" "${HAB_CACHE_SRC_PATH}/${pkg_dirname}"
```

Comment:
Maybe we can use the tar.gz download of sources: https://boringssl.googlesource.com/boringssl/+archive/3359.tar.gz ? This would make the plan simpler ^^

Reply:
oh snap I didn't know that worked